tests/test_force_https.py

```diff
@@ -0,0 +1,32 @@
+from tests.helpers import BaseApplicationTest, Config
+from dmutils import config, force_https
+from flask import Flask
+
+
+class TestForceHTTPSConfig(Config):
+    DM_HTTP_PROTO = "HTTPS"
+
+
+class TestForceHTTPS(BaseApplicationTest):
+    config_object = TestForceHTTPSConfig()
```

Comment:
In test_terms.py I just overrode the config var and everything got set up when I called `super`.  Doesn't that work here?

Otherwise, you might as well be inheriting from `object` because you're not really using anything from `BaseApplicationTest`